platforms/springboot/src/integration-test/java/io/hawt/springboot/HawtioSpringBootTestSupport.java

```diff
@@ -229,13 +244,13 @@ public TestPropertiesBuilder servletPath(String servletPath) {
             return this;
         }
 
-        public TestPropertiesBuilder managementContextPath(String managementContextPath) {
+        public TestPropertiesBuilder managemenBasePath(String managementContextPath) {
```

Comment:
Typo: `managemenBasePath` -> `managementBasePath`

Also, we should rename the method parameter and field names from `managementContextPath` to `managementBasePath`.